src/onboarding/onboarding.js

```diff
@@ -0,0 +1,929 @@
+const fetch = require('node-fetch');
+
+const { Headers } = fetch;
+
+/**
+ * Sends a request given an args array generated via lib functions in the same module.
+ *
+ * @param {function} request
+ * @param {function} fetchFunc
+ *
+ * @returns {Promise<void>} The result of the sent request
+ */
+async function sendRequest(request, fetchFunc = fetch) {
+    await fetchFunc(...request);
+}
+
+/**
+ * Returns the settlement account ID of a given DFSP's accounts on the hub.
+ *
+ * @param {array} hubAccounts Array of accounts registered to a specific DFSP on the hub
+ * @param {string} dfspCurrency Three-letter currency string, e.g. EUR
+ *
+ * @returns {number} settlementId
+ */
+const settlementIdFromHubAccounts = (hubAccounts, dfspCurrency) => hubAccounts
+    .filter((account) => account.currency === dfspCurrency.toUpperCase())
+    .find((account) => account.ledgerAccountType === 'SETTLEMENT')
+    .id;
+
+/**
+ * Returns an array with args to get accounts for a given DFSP associated with the hub.
+ *
+ * @param options
+ * @param {string} options.dfspName
+ * @param {string} options.authToken
+ * @param {string} options.hostCentralLedger
+ * @param {string} [options.baseCentralLedgerAdmin]
+ * @param {string} [options.fspiopSource]
+ * ß
+ * @returns {array} url, requestOptions
+ */
+function getHubAccounts({
```

Comment:
@KamuelaFranco ... should we call it getHubAccounts or getDfspAccounts, as looks like this function is getting accounts based on the dfsp name being passed

Reply:
Thank you, @sri-miriyala. I agree and have made the change.